src/Cursors/SvgCursor.ts

```diff
@@ -187,29 +210,50 @@ export class SVGCursor extends TouchlessCursor {
         this.cursorCanvas.style.opacity = opacity.toString();
     }
 
+    // Function: GetCurrentCursorRadius
+    // Used to set the radius of the cursor
     private GetCurrentCursorRadius(): number {
-        if (this.cursor) {
-            const radius = this.cursor.getAttribute('r');
-            if (!radius) {
-                return 0;
-            }
+        const radius = this.cursor?.getAttribute('r');
+        if (!radius) {
+            return 0;
+        }
+
+        const radiusAsNumber = parseFloat(radius);
```

Comment:
👀 `radius` could be undefined here - I think the original check for cursor and returning `undefined` is probably still sensible to do.

Reply:
The !radius should catch it if its undefined? (and doesn't matter if cursor is undefined due to the ? which would result in radius being undefined which is then caught here)

Reply:
You're right actually, but the new way you've done it is even better! Love it 👌